server/fleet/mdm.go

```diff
@@ -196,10 +196,10 @@ type ABMTermsUpdater interface {
 // MDMIdPAccount contains account information of a third-party IdP that can be
 // later used for MDM operations like creating local accounts.
 type MDMIdPAccount struct {
-	UUID     string
-	Username string
-	Fullname string
-	Email    string
+	UUID     string `db:"uuid"`
+	Username string `db:"username"`
+	Fullname string `db:"fullname"`
+	Email    string `db:"email"`
```

Comment:
So that we can pull a `mdm_idp_account` record into a struct to send to [`maybeAssociateHostMDMIdPWithScimUser`](https://github.com/fleetdm/fleet/pull/37327/changes#diff-ec797a071df046dfb849880d689b5dc274601d19626e17e2f61e6ce663adaea9R4445)